app/controllers/api/v1/search_results_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SearchResultsController < ApplicationController
```

Comment:
@Tuubz By renaming the controller, the upload will belong to `search_results` endpoint?  `api/v1/search_results/upload` 🤔

Reply:
No, only `http://0.0.0.0:3000/api/v1/upload` will be supported 🚀

Reply:
Ah ha...There is `resources :upload` 🆗